cmd/kubectl-directpv/info.go

```diff
@@ -185,7 +180,7 @@ func getInfo(ctx context.Context, args []string, quiet bool) error {
 
 		if driveCount == 0 {
 			t.AppendRow([]interface{}{
-				fmt.Sprintf("%s %s", color.RedString(dot), n),
+				fmt.Sprintf("%s %s", color.GreenString(dot), n),
```

Comment:
Can we keep the `red` string - It is to indicate that no drives are added yet to DirectPV

Reply:
`Red` seems error. I will change it to `Yellow`